Tensorflow not found

I'm trying to compile essentia with Gaia and Tensorflow support, with no luck.
Tensorflow is installed with `pip3 install --upgrade tensorflow`
It seems like Tensorflow is not found in the pkg-config search path...

`./waf configure --build-static --with-python --with-cpptests --with-examples --with-gaia --with-tensorflow

Setting top to                           : /home/dev/essentia 
Setting out to                           : /home/dev/essentia/build 
→ configuring the project in /home/dev/essentia
→ Building in release mode
Checking for 'g++' (C++ compiler)        : /usr/bin/g++ 
Checking for 'gcc' (C compiler)          : /usr/bin/gcc 
Checking for program 'pkg-config'        : /usr/bin/pkg-config 
Checking for 'eigen3'                    : yes 
Checking for 'libavcodec' >= 55.34.1     : yes 
Checking for 'libavformat'               : yes 
Checking for 'libavutil'                 : yes 
Checking for 'libavresample'             : yes 
Checking for 'samplerate'                : yes 
Checking for 'taglib' >= 1.9             : yes 
Checking for 'yaml-0.1'                  : yes 
Checking for 'fftw3f'                    : yes 
Checking for 'libchromaprint'            : yes 
Checking for 'gaia2'                     : yes 
Checking for 'tensorflow'                : not found 
The configuration failed
(complete log in /home/dev/essentia/build/config.log)
`

The relevant config.log section:
`----------------------------------------
Checking for 'tensorflow'
['/usr/bin/pkg-config', '--cflags', '--libs', 'tensorflow']
err: Package tensorflow was not found in the pkg-config search path.
Perhaps you should add the directory containing `tensorflow.pc'
to the PKG_CONFIG_PATH environment variable
No package 'tensorflow' found

not found`

Ok fresh Linux installation...
I've launched the ssetup script to make libs visible to pkg-config, but `./waf -v` results in:

`Build failed
 -> task in 'essentia_standard_pitchdemo' failed with exit status 1: 
	{task 139622116396624: cxxprogram standard_pitchdemo.cpp.5.o -> essentia_standard_pitchdemo}
['/usr/bin/g++', '-pthread', 'src/examples/standard_pitchdemo.cpp.5.o', '-o/home/dev/essentia/build/src/examples/essentia_standard_pitchdemo', '-Wl,-Bstatic', '-Lsrc', '-lessentia', '-Wl,-Bdynamic', '-L/usr/local/lib', '-L\\[QtCore\\]', '-L/usr/local/lib', '-lfftw3f', '-lavformat', '-lavcodec', '-lavutil', '-lavresample', '-lsamplerate', '-ltag', '-lyaml', '-lgaia2', '-lQtCore', '-lyaml', '-lchromaprint', '-lpywrap_tensorflow_internal', '-ltensorflow_framework']
 -> task in 'essentia_standard_vibrato' failed with exit status 1: 
	{task 139622116483536: cxxprogram standard_vibrato.cpp.8.o -> essentia_standard_vibrato}
['/usr/bin/g++', '-pthread', 'src/examples/standard_vibrato.cpp.8.o', '-o/home/dev/essentia/build/src/examples/essentia_standard_vibrato', '-Wl,-Bstatic', '-Lsrc', '-lessentia', '-Wl,-Bdynamic', '-L/usr/local/lib', '-L\\[QtCore\\]', '-L/usr/local/lib', '-lfftw3f', '-lavformat', '-lavcodec', '-lavutil', '-lavresample', '-lsamplerate', '-ltag', '-lyaml', '-lgaia2', '-lQtCore', '-lyaml', '-lchromaprint', '-lpywrap_tensorflow_internal', '-ltensorflow_framework']
 -> task in 'essentia_standard_mfcc' failed with exit status 1: 
	{task 139622116396112: cxxprogram standard_mfcc.cpp.3.o -> essentia_standard_mfcc}
['/usr/bin/g++', '-pthread', 'src/examples/standard_mfcc.cpp.3.o', '-o/home/dev/essentia/build/src/examples/essentia_standard_mfcc', '-Wl,-Bstatic', '-Lsrc', '-lessentia', '-Wl,-Bdynamic', '-L/usr/local/lib', '-L\\[QtCore\\]', '-L/usr/local/lib', '-lfftw3f', '-lavformat', '-lavcodec', '-lavutil', '-lavresample', '-lsamplerate', '-ltag', '-lyaml', '-lgaia2', '-lQtCore', '-lyaml', '-lchromaprint', '-lpywrap_tensorflow_internal', '-ltensorflow_framework']
 -> task in 'essentia_standard_fadedetection' failed with exit status 1: 
	{task 139622116395856: cxxprogram standard_fadedetection.cpp.2.o -> essentia_standard_fadedetection}
['/usr/bin/g++', '-pthread', 'src/examples/standard_fadedetection.cpp.2.o', '-o/home/dev/essentia/build/src/examples/essentia_standard_fadedetection', '-Wl,-Bstatic', '-Lsrc', '-lessentia', '-Wl,-Bdynamic', '-L/usr/local/lib', '-L\\[QtCore\\]', '-L/usr/local/lib', '-lfftw3f', '-lavformat', '-lavcodec', '-lavutil', '-lavresample', '-lsamplerate', '-ltag', '-lyaml', '-lgaia2', '-lQtCore', '-lyaml', '-lchromaprint', '-lpywrap_tensorflow_internal', '-ltensorflow_framework']
 -> task in 'essentia_standard_onsetrate' failed with exit status 1: 
	{task 139622116396368: cxxprogram standard_onsetrate.cpp.4.o -> essentia_standard_onsetrate}
['/usr/bin/g++', '-pthread', 'src/examples/standard_onsetrate.cpp.4.o', '-o/home/dev/essentia/build/src/examples/essentia_standard_onsetrate', '-Wl,-Bstatic', '-Lsrc', '-lessentia', '-Wl,-Bdynamic', '-L/usr/local/lib', '-L\\[QtCore\\]', '-L/usr/local/lib', '-lfftw3f', '-lavformat', '-lavcodec', '-lavutil', '-lavresample', '-lsamplerate', '-ltag', '-lyaml', '-lgaia2', '-lQtCore', '-lyaml', '-lchromaprint', '-lpywrap_tensorflow_internal', '-ltensorflow_framework']
 -> task in 'essentia_standard_hprmodel' failed with exit status 1: 
	{task 139622116484304: cxxprogram standard_hprmodel.cpp.11.o -> essentia_standard_hprmodel}
['/usr/bin/g++', '-pthread', 'src/examples/standard_hprmodel.cpp.11.o', '-o/home/dev/essentia/build/src/examples/essentia_standard_hprmodel', '-Wl,-Bstatic', '-Lsrc', '-lessentia', '-Wl,-Bdynamic', '-L/usr/local/lib', '-L\\[QtCore\\]', '-L/usr/local/lib', '-lfftw3f', '-lavformat', '-lavcodec', '-lavutil', '-lavresample', '-lsamplerate', '-ltag', '-lyaml', '-lgaia2', '-lQtCore', '-lyaml', '-lchromaprint', '-lpywrap_tensorflow_internal', '-ltensorflow_framework']
 -> task in 'essentia_standard_sprmodel' failed with exit status 1: 
	{task 139622116485328: cxxprogram standard_sprmodel.cpp.15.o -> essentia_standard_sprmodel}
['/usr/bin/g++', '-pthread', 'src/examples/standard_sprmodel.cpp.15.o', '-o/home/dev/essentia/build/src/examples/essentia_standard_sprmodel', '-Wl,-Bstatic', '-Lsrc', '-lessentia', '-Wl,-Bdynamic', '-L/usr/local/lib', '-L\\[QtCore\\]', '-L/usr/local/lib', '-lfftw3f', '-lavformat', '-lavcodec', '-lavutil', '-lavresample', '-lsamplerate', '-ltag', '-lyaml', '-lgaia2', '-lQtCore', '-lyaml', '-lchromaprint', '-lpywrap_tensorflow_internal', '-ltensorflow_framework']
 -> task in 'essentia_standard_pca' failed with exit status 1: 
	{task 139622116486480: cxxprogram standard_pca.cpp.20.o -> essentia_standard_pca}
['/usr/bin/g++', '-pthread', 'src/examples/standard_pca.cpp.20.o', '-o/home/dev/essentia/build/src/examples/essentia_standard_pca', '-Wl,-Bstatic', '-Lsrc', '-lessentia', '-Wl,-Bdynamic', '-L/usr/local/lib', '-L\\[QtCore\\]', '-L/usr/local/lib', '-lfftw3f', '-lavformat', '-lavcodec', '-lavutil', '-lavresample', '-lsamplerate', '-ltag', '-lyaml', '-lgaia2', '-lQtCore', '-lyaml', '-lchromaprint', '-lpywrap_tensorflow_internal', '-ltensorflow_framework']
 -> task in 'essentia_standard_predominantmask' failed with exit status 1: 
	{task 139622116484560: cxxprogram standard_predominantmask.cpp.12.o -> essentia_standard_predominantmask}
['/usr/bin/g++', '-pthread', 'src/examples/standard_predominantmask.cpp.12.o', '-o/home/dev/essentia/build/src/examples/essentia_standard_predominantmask', '-Wl,-Bstatic', '-Lsrc', '-lessentia', '-Wl,-Bdynamic', '-L/usr/local/lib', '-L\\[QtCore\\]', '-L/usr/local/lib', '-lfftw3f', '-lavformat', '-lavcodec', '-lavutil', '-lavresample', '-lsamplerate', '-ltag', '-lyaml', '-lgaia2', '-lQtCore', '-lyaml', '-lchromaprint', '-lpywrap_tensorflow_internal', '-ltensorflow_framework']
 -> task in 'essentia_standard_sinesubtraction' failed with exit status 1: 
	{task 139622116485072: cxxprogram standard_sinesubtraction.cpp.14.o -> essentia_standard_sinesubtraction}
['/usr/bin/g++', '-pthread', 'src/examples/standard_sinesubtraction.cpp.14.o', '-o/home/dev/essentia/build/src/examples/essentia_standard_sinesubtraction', '-Wl,-Bstatic', '-Lsrc', '-lessentia', '-Wl,-Bdynamic', '-L/usr/local/lib', '-L\\[QtCore\\]', '-L/usr/local/lib', '-lfftw3f', '-lavformat', '-lavcodec', '-lavutil', '-lavresample', '-lsamplerate', '-ltag', '-lyaml', '-lgaia2', '-lQtCore', '-lyaml', '-lchromaprint', '-lpywrap_tensorflow_internal', '-ltensorflow_framework']
 -> task in 'essentia_standard_stft' failed with exit status 1: 
	{task 139622116485840: cxxprogram standard_stft.cpp.17.o -> essentia_standard_stft}
['/usr/bin/g++', '-pthread', 'src/examples/standard_stft.cpp.17.o', '-o/home/dev/essentia/build/src/examples/essentia_standard_stft', '-Wl,-Bstatic', '-Lsrc', '-lessentia', '-Wl,-Bdynamic', '-L/usr/local/lib', '-L\\[QtCore\\]', '-L/usr/local/lib', '-lfftw3f', '-lavformat', '-lavcodec', '-lavutil', '-lavresample', '-lsamplerate', '-ltag', '-lyaml', '-lgaia2', '-lQtCore', '-lyaml', '-lchromaprint', '-lpywrap_tensorflow_internal', '-ltensorflow_framework']
 -> task in 'essentia_standard_spsmodel' failed with exit status 1: 
	{task 139622116485584: cxxprogram standard_spsmodel.cpp.16.o -> essentia_standard_spsmodel}
['/usr/bin/g++', '-pthread', 'src/examples/standard_spsmodel.cpp.16.o', '-o/home/dev/essentia/build/src/examples/essentia_standard_spsmodel', '-Wl,-Bstatic', '-Lsrc', '-lessentia', '-Wl,-Bdynamic', '-L/usr/local/lib', '-L\\[QtCore\\]', '-L/usr/local/lib', '-lfftw3f', '-lavformat', '-lavcodec', '-lavutil', '-lavresample', '-lsamplerate', '-ltag', '-lyaml', '-lgaia2', '-lQtCore', '-lyaml', '-lchromaprint', '-lpywrap_tensorflow_internal', '-ltensorflow_framework']
 -> task in 'essentia_standard_sinemodel' failed with exit status 1: 
	{task 139622116484816: cxxprogram standard_sinemodel.cpp.13.o -> essentia_standard_sinemodel}
['/usr/bin/g++', '-pthread', 'src/examples/standard_sinemodel.cpp.13.o', '-o/home/dev/essentia/build/src/examples/essentia_standard_sinemodel', '-Wl,-Bstatic', '-Lsrc', '-lessentia', '-Wl,-Bdynamic', '-L/usr/local/lib', '-L\\[QtCore\\]', '-L/usr/local/lib', '-lfftw3f', '-lavformat', '-lavcodec', '-lavutil', '-lavresample', '-lsamplerate', '-ltag', '-lyaml', '-lgaia2', '-lQtCore', '-lyaml', '-lchromaprint', '-lpywrap_tensorflow_internal', '-ltensorflow_framework']
 -> task in 'essentia_standard_humdetector' failed with exit status 1: 
	{task 139622116786640: cxxprogram standard_humdetector.cpp.24.o -> essentia_standard_humdetector}
['/usr/bin/g++', '-pthread', 'src/examples/standard_humdetector.cpp.24.o', '-o/home/dev/essentia/build/src/examples/essentia_standard_humdetector', '-Wl,-Bstatic', '-Lsrc', '-lessentia', '-Wl,-Bdynamic', '-L/usr/local/lib', '-L\\[QtCore\\]', '-L/usr/local/lib', '-lfftw3f', '-lavformat', '-lavcodec', '-lavutil', '-lavresample', '-lsamplerate', '-ltag', '-lyaml', '-lgaia2', '-lQtCore', '-lyaml', '-lchromaprint', '-lpywrap_tensorflow_internal', '-ltensorflow_framework']
 -> task in 'essentia_standard_harmonicmodel' failed with exit status 1: 
	{task 139622116483792: cxxprogram standard_harmonicmodel.cpp.9.o -> essentia_standard_harmonicmodel}
['/usr/bin/g++', '-pthread', 'src/examples/standard_harmonicmodel.cpp.9.o', '-o/home/dev/essentia/build/src/examples/essentia_standard_harmonicmodel', '-Wl,-Bstatic', '-Lsrc', '-lessentia', '-Wl,-Bdynamic', '-L/usr/local/lib', '-L\\[QtCore\\]', '-L/usr/local/lib', '-lfftw3f', '-lavformat', '-lavcodec', '-lavutil', '-lavresample', '-lsamplerate', '-ltag', '-lyaml', '-lgaia2', '-lQtCore', '-lyaml', '-lchromaprint', '-lpywrap_tensorflow_internal', '-ltensorflow_framework']
 -> task in 'essentia_standard_chromaprinter' failed with exit status 1: 
	{task 139622116486352: cxxprogram standard_chromaprinter.cpp.19.o -> essentia_standard_chromaprinter}
['/usr/bin/g++', '-pthread', 'src/examples/standard_chromaprinter.cpp.19.o', '-o/home/dev/essentia/build/src/examples/essentia_standard_chromaprinter', '-Wl,-Bstatic', '-Lsrc', '-lessentia', '-Wl,-Bdynamic', '-L/usr/local/lib', '-L\\[QtCore\\]', '-L/usr/local/lib', '-lfftw3f', '-lavformat', '-lavcodec', '-lavutil', '-lavresample', '-lsamplerate', '-ltag', '-lyaml', '-lgaia2', '-lQtCore', '-lyaml', '-lchromaprint', '-lpywrap_tensorflow_internal', '-ltensorflow_framework']
 -> task in 'essentia_standard_discontinuitydetector' failed with exit status 1: 
	{task 139622116486736: cxxprogram standard_discontinuitydetector.cpp.21.o -> essentia_standard_discontinuitydetector}
['/usr/bin/g++', '-pthread', 'src/examples/standard_discontinuitydetector.cpp.21.o', '-o/home/dev/essentia/build/src/examples/essentia_standard_discontinuitydetector', '-Wl,-Bstatic', '-Lsrc', '-lessentia', '-Wl,-Bdynamic', '-L/usr/local/lib', '-L\\[QtCore\\]', '-L/usr/local/lib', '-lfftw3f', '-lavformat', '-lavcodec', '-lavutil', '-lavresample', '-lsamplerate', '-ltag', '-lyaml', '-lgaia2', '-lQtCore', '-lyaml', '-lchromaprint', '-lpywrap_tensorflow_internal', '-ltensorflow_framework']
 -> task in 'essentia_standard_saturationdetector' failed with exit status 1: 
	{task 139622116787152: cxxprogram standard_saturationdetector.cpp.26.o -> essentia_standard_saturationdetector}
['/usr/bin/g++', '-pthread', 'src/examples/standard_saturationdetector.cpp.26.o', '-o/home/dev/essentia/build/src/examples/essentia_standard_saturationdetector', '-Wl,-Bstatic', '-Lsrc', '-lessentia', '-Wl,-Bdynamic', '-L/usr/local/lib', '-L\\[QtCore\\]', '-L/usr/local/lib', '-lfftw3f', '-lavformat', '-lavcodec', '-lavutil', '-lavresample', '-lsamplerate', '-ltag', '-lyaml', '-lgaia2', '-lQtCore', '-lyaml', '-lchromaprint', '-lpywrap_tensorflow_internal', '-ltensorflow_framework']
 -> task in 'essentia_standard_stochasticmodel' failed with exit status 1: 
	{task 139622116486096: cxxprogram standard_stochasticmodel.cpp.18.o -> essentia_standard_stochasticmodel}
['/usr/bin/g++', '-pthread', 'src/examples/standard_stochasticmodel.cpp.18.o', '-o/home/dev/essentia/build/src/examples/essentia_standard_stochasticmodel', '-Wl,-Bstatic', '-Lsrc', '-lessentia', '-Wl,-Bdynamic', '-L/usr/local/lib', '-L\\[QtCore\\]', '-L/usr/local/lib', '-lfftw3f', '-lavformat', '-lavcodec', '-lavutil', '-lavresample', '-lsamplerate', '-ltag', '-lyaml', '-lgaia2', '-lQtCore', '-lyaml', '-lchromaprint', '-lpywrap_tensorflow_internal', '-ltensorflow_framework']
 -> task in 'essentia_standard_spectralcontrast' failed with exit status 1: 
	{task 139622116483280: cxxprogram standard_spectralcontrast.cpp.7.o -> essentia_standard_spectralcontrast}
['/usr/bin/g++', '-pthread', 'src/examples/standard_spectralcontrast.cpp.7.o', '-o/home/dev/essentia/build/src/examples/essentia_standard_spectralcontrast', '-Wl,-Bstatic', '-Lsrc', '-lessentia', '-Wl,-Bdynamic', '-L/usr/local/lib', '-L\\[QtCore\\]', '-L/usr/local/lib', '-lfftw3f', '-lavformat', '-lavcodec', '-lavutil', '-lavresample', '-lsamplerate', '-ltag', '-lyaml', '-lgaia2', '-lQtCore', '-lyaml', '-lchromaprint', '-lpywrap_tensorflow_internal', '-ltensorflow_framework']
 -> task in 'essentia_standard_hpsmodel' failed with exit status 1: 
	{task 139622116484048: cxxprogram standard_hpsmodel.cpp.10.o -> essentia_standard_hpsmodel}
['/usr/bin/g++', '-pthread', 'src/examples/standard_hpsmodel.cpp.10.o', '-o/home/dev/essentia/build/src/examples/essentia_standard_hpsmodel', '-Wl,-Bstatic', '-Lsrc', '-lessentia', '-Wl,-Bdynamic', '-L/usr/local/lib', '-L\\[QtCore\\]', '-L/usr/local/lib', '-lfftw3f', '-lavformat', '-lavcodec', '-lavutil', '-lavresample', '-lsamplerate', '-ltag', '-lyaml', '-lgaia2', '-lQtCore', '-lyaml', '-lchromaprint', '-lpywrap_tensorflow_internal', '-ltensorflow_framework']
 -> task in 'essentia_standard_welch' failed with exit status 1: 
	{task 139622116787664: cxxprogram standard_welch.cpp.28.o -> essentia_standard_welch}
['/usr/bin/g++', '-pthread', 'src/examples/standard_welch.cpp.28.o', '-o/home/dev/essentia/build/src/examples/essentia_standard_welch', '-Wl,-Bstatic', '-Lsrc', '-lessentia', '-Wl,-Bdynamic', '-L/usr/local/lib', '-L\\[QtCore\\]', '-L/usr/local/lib', '-lfftw3f', '-lavformat', '-lavcodec', '-lavutil', '-lavresample', '-lsamplerate', '-ltag', '-lyaml', '-lgaia2', '-lQtCore', '-lyaml', '-lchromaprint', '-lpywrap_tensorflow_internal', '-ltensorflow_framework']
 -> task in 'essentia_standard_gapsdetector' failed with exit status 1: 
	{task 139622116786384: cxxprogram standard_gapsdetector.cpp.23.o -> essentia_standard_gapsdetector}
['/usr/bin/g++', '-pthread', 'src/examples/standard_gapsdetector.cpp.23.o', '-o/home/dev/essentia/build/src/examples/essentia_standard_gapsdetector', '-Wl,-Bstatic', '-Lsrc', '-lessentia', '-Wl,-Bdynamic', '-L/usr/local/lib', '-L\\[QtCore\\]', '-L/usr/local/lib', '-lfftw3f', '-lavformat', '-lavcodec', '-lavutil', '-lavresample', '-lsamplerate', '-ltag', '-lyaml', '-lgaia2', '-lQtCore', '-lyaml', '-lchromaprint', '-lpywrap_tensorflow_internal', '-ltensorflow_framework']
 -> task in 'essentia_streaming_humdetector' failed with exit status 1: 
	{task 139622116787920: cxxprogram streaming_humdetector.cpp.29.o -> essentia_streaming_humdetector}
['/usr/bin/g++', '-pthread', 'src/examples/streaming_humdetector.cpp.29.o', '-o/home/dev/essentia/build/src/examples/essentia_streaming_humdetector', '-Wl,-Bstatic', '-Lsrc', '-lessentia', '-Wl,-Bdynamic', '-L/usr/local/lib', '-L\\[QtCore\\]', '-L/usr/local/lib', '-lfftw3f', '-lavformat', '-lavcodec', '-lavutil', '-lavresample', '-lsamplerate', '-ltag', '-lyaml', '-lgaia2', '-lQtCore', '-lyaml', '-lchromaprint', '-lpywrap_tensorflow_internal', '-ltensorflow_framework']
 -> task in 'essentia_standard_crosssimilaritymatrix' failed with exit status 1: 
	{task 139622116788176: cxxprogram standard_crosssimilaritymatrix.cpp.30.o -> essentia_standard_crosssimilaritymatrix}
['/usr/bin/g++', '-pthread', 'src/examples/standard_crosssimilaritymatrix.cpp.30.o', '-o/home/dev/essentia/build/src/examples/essentia_standard_crosssimilaritymatrix', '-Wl,-Bstatic', '-Lsrc', '-lessentia', '-Wl,-Bdynamic', '-L/usr/local/lib', '-L\\[QtCore\\]', '-L/usr/local/lib', '-lfftw3f', '-lavformat', '-lavcodec', '-lavutil', '-lavresample', '-lsamplerate', '-ltag', '-lyaml', '-lgaia2', '-lQtCore', '-lyaml', '-lchromaprint', '-lpywrap_tensorflow_internal', '-ltensorflow_framework']
 -> task in 'essentia_standard_extractor_la-cupula' failed with exit status 1: 
	{task 139622116486992: cxxprogram standard_extractor_la-cupula.cpp.22.o -> essentia_standard_extractor_la-cupula}
['/usr/bin/g++', '-pthread', 'src/examples/standard_extractor_la-cupula.cpp.22.o', '-o/home/dev/essentia/build/src/examples/essentia_standard_extractor_la-cupula', '-Wl,-Bstatic', '-Lsrc', '-lessentia', '-Wl,-Bdynamic', '-L/usr/local/lib', '-L\\[QtCore\\]', '-L/usr/local/lib', '-lfftw3f', '-lavformat', '-lavcodec', '-lavutil', '-lavresample', '-lsamplerate', '-ltag', '-lyaml', '-lgaia2', '-lQtCore', '-lyaml', '-lchromaprint', '-lpywrap_tensorflow_internal', '-ltensorflow_framework']
 -> task in 'essentia_standard_loudnessebur128_double_input' failed with exit status 1: 
	{task 139622116786896: cxxprogram standard_loudnessebur128_double_input.cpp.25.o -> essentia_standard_loudnessebur128_double_input}
['/usr/bin/g++', '-pthread', 'src/examples/standard_loudnessebur128_double_input.cpp.25.o', '-o/home/dev/essentia/build/src/examples/essentia_standard_loudnessebur128_double_input', '-Wl,-Bstatic', '-Lsrc', '-lessentia', '-Wl,-Bdynamic', '-L/usr/local/lib', '-L\\[QtCore\\]', '-L/usr/local/lib', '-lfftw3f', '-lavformat', '-lavcodec', '-lavutil', '-lavresample', '-lsamplerate', '-ltag', '-lyaml', '-lgaia2', '-lQtCore', '-lyaml', '-lchromaprint', '-lpywrap_tensorflow_internal', '-ltensorflow_framework']
 -> task in 'essentia_standard_snr' failed with exit status 1: 
	{task 139622116787408: cxxprogram standard_snr.cpp.27.o -> essentia_standard_snr}
['/usr/bin/g++', '-pthread', 'src/examples/standard_snr.cpp.27.o', '-o/home/dev/essentia/build/src/examples/essentia_standard_snr', '-Wl,-Bstatic', '-Lsrc', '-lessentia', '-Wl,-Bdynamic', '-L/usr/local/lib', '-L\\[QtCore\\]', '-L/usr/local/lib', '-lfftw3f', '-lavformat', '-lavcodec', '-lavutil', '-lavresample', '-lsamplerate', '-ltag', '-lyaml', '-lgaia2', '-lQtCore', '-lyaml', '-lchromaprint', '-lpywrap_tensorflow_internal', '-ltensorflow_framework']
 -> task in 'essentia_standard_coversongsimilarity' failed with exit status 1: 
	{task 139622116788688: cxxprogram standard_coversongsimilarity.cpp.32.o -> essentia_standard_coversongsimilarity}
['/usr/bin/g++', '-pthread', 'src/examples/standard_coversongsimilarity.cpp.32.o', '-o/home/dev/essentia/build/src/examples/essentia_standard_coversongsimilarity', '-Wl,-Bstatic', '-Lsrc', '-lessentia', '-Wl,-Bdynamic', '-L/usr/local/lib', '-L\\[QtCore\\]', '-L/usr/local/lib', '-lfftw3f', '-lavformat', '-lavcodec', '-lavutil', '-lavresample', '-lsamplerate', '-ltag', '-lyaml', '-lgaia2', '-lQtCore', '-lyaml', '-lchromaprint', '-lpywrap_tensorflow_internal', '-ltensorflow_framework']
 -> task in 'essentia_standard_chromacrosssimilarity' failed with exit status 1: 
	{task 139622116788432: cxxprogram standard_chromacrosssimilarity.cpp.31.o -> essentia_standard_chromacrosssimilarity}
['/usr/bin/g++', '-pthread', 'src/examples/standard_chromacrosssimilarity.cpp.31.o', '-o/home/dev/essentia/build/src/examples/essentia_standard_chromacrosssimilarity', '-Wl,-Bstatic', '-Lsrc', '-lessentia', '-Wl,-Bdynamic', '-L/usr/local/lib', '-L\\[QtCore\\]', '-L/usr/local/lib', '-lfftw3f', '-lavformat', '-lavcodec', '-lavutil', '-lavresample', '-lsamplerate', '-ltag', '-lyaml', '-lgaia2', '-lQtCore', '-lyaml', '-lchromaprint', '-lpywrap_tensorflow_internal', '-ltensorflow_framework']
 -> task in 'essentia_standard_rhythmtransform' failed with exit status 1: 
	{task 139622116396880: cxxprogram standard_rhythmtransform.cpp.6.o -> essentia_standard_rhythmtransform}
['/usr/bin/g++', '-pthread', 'src/examples/standard_rhythmtransform.cpp.6.o', '-o/home/dev/essentia/build/src/examples/essentia_standard_rhythmtransform', '-Wl,-Bstatic', '-Lsrc', '-lessentia', '-Wl,-Bdynamic', '-L/usr/local/lib', '-L\\[QtCore\\]', '-L/usr/local/lib', '-lfftw3f', '-lavformat', '-lavcodec', '-lavutil', '-lavresample', '-lsamplerate', '-ltag', '-lyaml', '-lgaia2', '-lQtCore', '-lyaml', '-lchromaprint', '-lpywrap_tensorflow_internal', '-ltensorflow_framework']
 -> task in 'essentia_standard_beatsmarker' failed with exit status 1: 
	{task 139622116395600: cxxprogram standard_beatsmarker.cpp.1.o -> essentia_standard_beatsmarker}
['/usr/bin/g++', '-pthread', 'src/examples/standard_beatsmarker.cpp.1.o', '-o/home/dev/essentia/build/src/examples/essentia_standard_beatsmarker', '-Wl,-Bstatic', '-Lsrc', '-lessentia', '-Wl,-Bdynamic', '-L/usr/local/lib', '-L\\[QtCore\\]', '-L/usr/local/lib', '-lfftw3f', '-lavformat', '-lavcodec', '-lavutil', '-lavresample', '-lsamplerate', '-ltag', '-lyaml', '-lgaia2', '-lQtCore', '-lyaml', '-lchromaprint', '-lpywrap_tensorflow_internal', '-ltensorflow_framework']`


It seems it still can't find Tensorflow .so

`/usr/local/lib/libpywrap_tensorflow_internal.so: undefined reference to `PyList_Sort'
/usr/local/lib/libpywrap_tensorflow_internal.so: undefined reference to `PyType_IsSubtype'
/usr/local/lib/libpywrap_tensorflow_internal.so: undefined reference to `PySys_WriteStdout'
/usr/local/lib/libpywrap_tensorflow_internal.so: undefined reference to `PyObject_HasAttrString'
/usr/local/lib/libpywrap_tensorflow_internal.so: undefined reference to `PyDict_Keys'
/usr/local/lib/libpywrap_tensorflow_internal.so: undefined reference to `PyErr_Clear'
/usr/local/lib/libpywrap_tensorflow_internal.so: undefined reference to `PyErr_SetObject'
/usr/local/lib/libpywrap_tensorflow_internal.so: undefined reference to `PyNumber_Long'
/usr/local/lib/libpywrap_tensorflow_internal.so: undefined reference to `PyFloat_FromDouble'
/usr/local/lib/libpywrap_tensorflow_internal.so: undefined reference to `PyLong_AsLongLong'
/usr/local/lib/libpywrap_tensorflow_internal.so: undefined reference to `PyErr_Restore'
/usr/local/lib/libpywrap_tensorflow_internal.so: undefined reference to `PyTuple_Type'
/usr/local/lib/libpywrap_tensorflow_internal.so: undefined reference to `PyErr_Occurred'
/usr/local/lib/libpywrap_tensorflow_internal.so: undefined reference to `PyObject_CallMethodObjArgs'
/usr/local/lib/libpywrap_tensorflow_internal.so: undefined reference to `PyUnicode_FromFormat'
/usr/local/lib/libpywrap_tensorflow_internal.so: undefined reference to `PyExc_OverflowError'
/usr/local/lib/libpywrap_tensorflow_internal.so: undefined reference to `PyErr_SetString'
/usr/local/lib/libpywrap_tensorflow_internal.so: undefined reference to `PyList_Append'
`

What am I missing?

--- a/src/3rdparty/tensorflow/setup_from_python.sh
+++ b/src/3rdparty/tensorflow/setup_from_python.sh
@@ -6,7 +6,7 @@ PYTHON=python3
 
 # config
 MODE=python
-CONTEXT=/usr/local/  # run as sudo if the context directoryt is not owned
+CONTEXT=/usr/local/  # run as sudo if the context directory is not owned
 
 # setup
 $PYTHON setup_tensorflow.py -m $MODE -c $CONTEXT

--- a/src/3rdparty/tensorflow/setup_tensorflow.py
+++ b/src/3rdparty/tensorflow/setup_tensorflow.py
@@ -74,18 +74,27 @@ if __name__ == "__main__":
         # create symbolic links fo the libraries
         print('creating symbolic links...')
 
-        src = join(tf_dir, 'libtensorflow_framework.so.1')
-        tgt = join(context, 'lib', 'libtensorflow_framework.so')
-        call(['ln', '-sf', src, tgt])
-        call(['ln', '-sf', 'libtensorflow_framework.so', join(context, 'lib', 'libtensorflow_framework.so.1')])
+        libtensorflow = 'tensorflow_framework'
+        pywrap_tensorflow_internal = 'pywrap_tensorflow_internal'
 
-        src = join(tf_dir, 'python', '_pywrap_tensorflow_internal.so')
-        tgt = join(context, 'lib', 'libpywrap_tensorflow_internal.so')
+        libtensorflow_name = 'lib{}.so.{}'.format(libtensorflow, major_version)
+        src = join(tf_dir, libtensorflow_name)
+        tgt = join(context, 'lib', libtensorflow_name)
         call(['ln', '-sf', src, tgt])
-        call(['ln', '-sf', 'libpywrap_tensorflow_internal.so', join(context, 'lib', '_pywrap_tensorflow_internal.so')])
 
-        libs = ('-lpywrap_tensorflow_internal '
-                '-ltensorflow_framework')
+        pywrap_tensorflow_name = '_{}.so'.format(pywrap_tensorflow_internal)
+        src = join(tf_dir, 'python', pywrap_tensorflow_name)
+        tgt = join(context, 'lib', pywrap_tensorflow_name)
+        call(['ln', '-sf', src, tgt])
+
+        # add also symbolic links with standarized library names
+        tgt = join(context, 'lib', 'lib{}.so'.format(libtensorflow))
+        call(['ln', '-sf', libtensorflow_name, tgt])
+
+        tgt = join(context, 'lib', 'lib{}.so'.format(pywrap_tensorflow_internal))
+        call(['ln', '-sf', pywrap_tensorflow_name, tgt])
+
+        libs = ('-l{} -l{}'.format(pywrap_tensorflow_internal, libtensorflow))
 
         # copy headers to the context dir
         include_dir = join(context, 'include', 'tensorflow', 'c')

